### ISSUE TITLE
Discord /me uses _underscores_ to italicize, not *asterisks*

### ISSUE THREAD

**Issue by digitalcircuit** [sm00th/bitlbee-discord#26]
Discord now uses underscores to indicate ```/me``` messages, e.g. ```/me does a thing``` → ```_does a thing_```.  It might still be useful to translate ```*does a thing*``` as both produce italic text.

For example, typing the following:
```
/me sends a message from web-client, with Firefox Developer tools open...
```
Results in:
![Using a me command](https://dl.dropbox.com/u/839888/Hosting/Utilities/Development/Bitlbee/Plugins/bitlbee-discord/Using%20a%20me%20command.png#v2 )

Whenever ```incoming_me_translation``` is on, these messages should be translated back, e.g.:
```
-*- discord_nick does a thing
```

*Aside, handling Discord formatting to/from mIRC codes would be a neat future addition*

**Comment by sm00th** [comment]
Fixed in 710d3f9.

Not sure what do you mean by mIRC codes, but probably not going to happen just like any other client-specific stuff.

**Comment by digitalcircuit** [comment]
I've explained the mIRC codes in issue #28.  Though technically mIRC-specific, most IRC clients that I've heard of implement it now.  The Bitlbee OTR plugin uses mIRC codes to indicate status, too.

Thanks for getting these features implemented!
